pkg/activator/handler/handler_test.go

```diff
@@ -242,17 +242,17 @@ func TestActivationHandlerProxyHeader(t *testing.T) {
 	req.Header.Set(activator.RevisionHeaderNamespace, testNamespace)
 	req.Header.Set(activator.RevisionHeaderName, testRevName)
 
-	// set up config store to populate context
+	// Set up config store to populate context.
 	configStore := setupConfigStore(t, logging.FromContext(ctx))
 	ctx = configStore.ToContext(req.Context())
 	handler.ServeHTTP(writer, req.WithContext(ctx))
 
 	select {
 	case httpReq := <-interceptCh:
 		if got := httpReq.Header.Get(network.ProxyHeaderName); got != activator.Name {
-			t.Errorf("Header '%s' does not have the expected value. Want = '%s', got = '%s'.", network.ProxyHeaderName, activator.Name, got)
+			t.Errorf("Header %q = %q, want:  %q", network.ProxyHeaderName, got, activator.Name)
 		}
-	case <-time.After(5 * time.Second):
+	case <-time.After(1 * time.Second):
```

Comment:
nit: this change seems irrelevant to the description of this PR.

Reply:
Added a description of this change :)